skel-usb.c

```diff
@@ -38,6 +38,7 @@
 
 #include "skel-usb.h"
 
+unsigned char update_calib;
```

Comment:
... I told you already to NOT add any global variables ... 
Add a function which will _SET_ to one the variable when needed.
Yes it looks a bit "overkill" but this force the code to only set this variable.